packages/gatsby-source-nacelle/nacelle-fragments/Content.graphql

```diff
@@ -0,0 +1,81 @@
+fragment Content on Content {
```

Comment:
`nacelle-fragments` as a folder name seems kind of redundant since it's in a `nacelle` project. What about `graphql/fragments` or just `gql-fragments`?

Reply:
Yeah, `gql-fragments` is fine.